mods/ra/rules/defaults.yaml

```diff
@@ -648,10 +658,17 @@
 		WaterSequence: water
 	Parachutable:
 		KilledOnImpassableTerrain: false
-		ParachuteSequence: parach
-		ParachuteOpenSequence: open
-		ParachuteIdleSequence: idle
 	Passenger:
 	CustomSelectionSize:
 		CustomBounds: 20,20
+	WithParachute:
+		UpgradeTypes: parachute
+		UpgradeMinEnabledLevel: 1
+		Image: parach
+		Sequence: idle
+		OpeningSequence: open
+		ShadowImage: parach-shadow
+		ShadowSequence: idle
+	UpgradeManager:
```

Comment:
Something implies me this was caused by auto-upgrading - `^Soldier` inherits both two required traits from `^ExistsInWorld`.